packages/keyring-eth-qr/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
```

Comment:
Should we add a reference to this initial PR ?

Reply:
Right, good catch - added in [5a3b4bc](https://github.com/MetaMask/accounts/pull/60/commits/5a3b4bced00074ec598d5541b74e9084a6123ce2)